src/platform/sdl/main.c

```diff
@@ -47,6 +47,7 @@ static void _loadState(struct mCoreThread* thread) {
 int main(int argc, char** argv) {
 #ifdef _WIN32
 	AttachConsole(ATTACH_PARENT_PROCESS);
+	freopen("CONOUT$", "w", stdout);
```

Comment:
If this is being done here, should it be does in qt/main.cpp too?

Reply:
> You do not appear to have reverted it. Did you forget to commit the change?

Yes I forgot. Done.

I also removed the call to `FreeConsole`. It's not needed actually and `mArgumentsDeinit` was the wrong place.



> If this is being done here, should it be does in qt/main.cpp too?

It's only necessary in `mgba-sdl.exe` because it uses `printf` and it doesn't work work in a GUI app. It only works in a console app unless you do `AttachConsole`.